_studio/shared/include/mfx_config.h

```diff
@@ -85,8 +85,6 @@
     #endif // #if !defined(ANDROID)
```

Comment:
Remove MFX_ENABLE_MJPEG_VIDEO_DECODE and MFX_ENABLE_MJPEG_VIDEO_ENCODE from lines 64-65 above!